pages/linux/shred.md

```diff
@@ -0,0 +1,15 @@
+# shred
+
+> Shred - overwrite a file to hide its contents, and optionally delete it
+
+- Overwrite the specified FILE(s) repeatedly, in order to make it harder for even very expensive hardware probing to recover the data.
+
+`shred {{file}}`
+
+- overwrite the file with zeros
+
+`shred -z {{file}}`
+
+- amount of iterations
```

Comment:
Please use an action phrase for the description (e.g. "Overwrite the file a specified number of times")